website/docs/Queries.md

```diff
@@ -127,7 +127,7 @@ getByPlaceholderText(
   options?: {
     exact?: boolean;
     normalizer?: (text: string) => string;
-    includeHiddenElements?: boolean;
+    hidden?: boolean;
```

Comment:
why this change? includeHiddenElements is the naming we want to promote. But maybe we should put both in the doc?

Reply:
Oops unintended change due to rebase, fixed it